oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -56,6 +57,7 @@
 
   private static final long serialVersionUID = 4556936364828217687L;
   private static final long MINIMUM_TOKEN_MILLISECONDS = 60000L * 5L;
+  private static final Map<String, List<String>> EMPTY_EXTRA_HEADERS = Collections.emptyMap();
```

Comment:
will this be added to by subclasses? I'm not quite following but it looks like it might be; but this is immutable.

Reply:
`addQuotaProjectIdToRequestMetadata` returns a new `ImmutableMap` so nothing should be added to this (nor do we want anyone to).